bats/tests/containers/factory-reset.bats

```diff
@@ -86,57 +254,106 @@ check_installation() {
+    local assert=assert
+    local refute=refute
+
+    if [ "${1-}" == "before" ]; then
+        assert=refute
+        refute=assert
+    fi
     # Check if ./rd/bin was removed from the path
-    if is_unix; then
-        # TODO add check for config.fish
-        env_profiles=(
-            "$HOME/.bashrc"
-            "$HOME/.zshrc"
-            "$HOME/.cshrc"
-            "$HOME/.tcshrc"
-        )
-        for candidate in .bash_profile .bash_login .profile; do
-            if [ -e "$HOME/$candidate" ]; then
-                env_profiles+=("$HOME/$candidate")
-                # Only the first candidate that exists will be modified
-                if [ "${1-}" = "before" ]; then
-                    break
-                fi
+    # TODO add check for config.fish
+    env_profiles=(
+        "$HOME/.bashrc"
+        "$HOME/.zshrc"
+        "$HOME/.cshrc"
+        "$HOME/.tcshrc"
+    )
+    for candidate in .bash_profile .bash_login .profile; do
+        if [ -e "$HOME/$candidate" ]; then
+            env_profiles+=("$HOME/$candidate")
+            # Only the first candidate that exists will be modified
+            if [ "${assert}" = "refute" ]; then
+                break
             fi
-        done
-    fi
+        fi
+    done
 
     for profile in "${env_profiles[@]}"; do
         echo "$assert that $profile does not add ~/.rd/bin to the PATH"
         # cshrc: setenv PATH "/Users/jan/.rd/bin"\:"$PATH"
         # posix: export PATH="/Users/jan/.rd/bin:$PATH"
         run grep "PATH.\"$HOME/.rd/bin" "$profile"
-        ${assert}_failure
+        "${assert}"_failure
     done
+}
 
+check_rd_context() {
+    local assert=assert
+    local refute=refute
+
+    if [ "${1-}" == "before" ]; then
+        assert=refute
+        refute=assert
+    fi
     # Check if the rancher-desktop docker context has been removed
     if using_docker; then
-        if is_unix; then
-            echo "$assert that the docker context rancher-desktop does not exist"
-            run grep -r rancher-desktop "$HOME/.docker/contexts/meta"
-            ${assert}_failure
-        fi
+        echo "$assert that the docker context rancher-desktop does not exist"
+        run grep -r rancher-desktop "$HOME/.docker/contexts/meta"
+        "${assert}"_failure
     fi
+}
+
+check_lima() {
+    local assert=assert
+    local refute=refute
 
+    if [ "${1-}" == "before" ]; then
+        assert=refute
+        refute=assert
+    fi
     # Check if VM was killed
-    if is_unix; then
-        run limactl ls
-        ${assert}_output --partial "No instance found"
+    run limactl ls
+    "${assert}"_output --partial "No instance found"
+}
+
+check_WSL() {
+    local assert=assert
+    local refute=refute
+
+    if [ "${1-}" == "before" ]; then
+        assert=refute
+        refute=assert
     fi
+    # Check if rancher-desktop WSL distros are deleted on Windows
+    run powershell.exe -c "wsl.exe --list"
+    "${refute}"_line --partial "rancher-desktop-data"
```

Comment:
Why are you using `--partial`?

The only reason I can think of would be to allow the tester to make the distro the default, in which case the line would be `rancher-desktop-data (Default)`. But why should we expect that when running this test; it would be a misconfiguration from my point of view.

```suggestion
    "${refute}"_line "rancher-desktop-data"
```